app/controllers/images_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ImagesController < ApplicationController
+  before_filter :convert_photo, only: [:create, :update]
+
+  def create
+    user = User.find(params[:user_id])
+    image = user.images.new(image_params)
+    image.user_id = current_user.id
+    if image.save
+      render_json_message(:ok, message: "Image successfully added!")
+    else
+      render_json_message(:forbidden, errors: user.errors.full_messages)
+    end
+  end
+
+  def update
+    image = Image.find(params[:id])
+    if image.update(image_params)
+      render_json_message(:ok, message: "Image successfully updated!")
+    else
+      render_json_message(:forbidden, errors: user.errors.full_messages)
+    end
+  end
+
+  def convert_photo
+    return if params[:user].blank?
+    params[:user][:photo] = FileUploadUtils.convert_base64(params[:user][:photo])
+  end
+
+  def destroy
+    delete_params[:photo_ids].map { |i| i.to_i }
```

Comment:
Add a check in the backend to make sure that the current_user.id matches the image.user_id before allowing them to delete the picture!